prov/efa/Makefile.include

```diff
@@ -73,10 +73,31 @@ _efa_headers = \
 	prov/efa/src/rxr/rxr_atomic.h \
 	prov/efa/src/rxr/rdm_proto_v4.h
 
+
+if ENABLE_EFA_UNIT_TEST
+check_PROGRAMS += prov/efa/test/efa_unit_test
+TESTS += prov/efa/test/efa_unit_test
+nodist_prov_efa_test_efa_unit_test_SOURCES = \
+	prov/efa/test/efa_unit_tests.h \
+	prov/efa/test/efa_unit_tests.c \
+	prov/efa/test/rdma_core_mocks.c
+
+prov_efa_test_efa_unit_test_LDADD = $(efatest_LIBS) $(linkback)
+prov_efa_test_efa_unit_test_CPPFLAGS = $(cmocka_CPPFLAGS) -I$(top_srcdir)/include -pthread -I$(top_srcdir)/prov/efa/src -I$(top_srcdir)/prov/efa/src/rxr -I$(top_srcdir)/prov/efa/test
+
+prov_efa_test_efa_unit_test_LDFLAGS = $(efa_LDFLAGS) $(cmocka_LDFLAGS) -Wl,--wrap=ibv_create_ah \
+																		-Wl,--wrap=efadv_query_ah
+
+efa_CPPFLAGS += -I$(top_srcdir)/prov/efa/test \
```

Comment:
If you move this to before `prov_efa_test_efa_unit_test_CPPFLAGS`, then `prov_efa_test_efa_unit_test_CPPFLAGS` will be able to use `efa_CPPFLAGS`.